spec/models/user_spec.rb

```diff
@@ -1,5 +1,73 @@
-require 'rails_helper'
+require "rails_helper"
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "enum for role" do
+    it {is_expected.to define_enum_for(:role).with_values(admin: 0, customer: 1)}
+  end
+
+  describe "associations" do
+    it {is_expected.to have_many(:orders).dependent(:destroy)}
+    it {is_expected.to have_many(:comments).dependent(:destroy)}
+  end
+
+  describe "validates" do
```

Comment:
Tìm hiểu về `describe, context, it`

Reply:
Describe thì nên dùng khi mô tả các method hoặc function được viết test. Còn context thì mô tả các trường hợp xảy ra đối với method hoặc function đó. It được sử dụng để định nghĩa một "Ví dụ" hay nói cách khác là dùng để định nghĩa 1 test case.